bin/ext_service/create_party_leaders_in_db.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import emission.core.get_database as edb
+
+if __name__ == '__main__':
```

Comment:
For maintainability, this should call `create_party_leaders()`.
But if everything else passes, I will let this slide and let you clean up later.